docs/source/trading-calendars.rst

```diff
@@ -0,0 +1,172 @@
+Trading Calendars
+-----------------
+
+What is a Trading Calendar?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A trading calendar represents the timing information of a single market exchange. The timing information is made up of two parts: sessions, and opens/closes. This is represented by the Zipline :class:`~zipline.utils.calendars.trading_calendar.TradingCalendar` class, and is used as the parent class for all new ``TradingCalendar`` s.
+
+A session represents a contiguous set of minutes, and has a label that is midnight UTC. It is important to note that a session label should not be considered a specific point in time, and that midnight UTC is just being used for convenience.
+
+Why Should You Care About Trading Calendars?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's say you want to buy a share of some equity on Tuesday, and then sell it on Saturday. If the exchange in which you're trading that equity is not open on Saturday, then in reality it would not be possible to trade that equity at that time, and you would have to wait until some other number of days past Saturday. It would then be unreasonable to allow your trading algorithm to also place a trade on Saturday.
```

Comment:
Maybe we can reword the last line? 

"Since you wouldn't be able to place the trade in reality, it would also be unreasonable for your backtest to place a trade on Saturday."